File: src/main/java/com/swp391/koibe/repositories/KoiImageRepository.java
package com.swp391.koibe.repositories;

import com.swp391.koibe.models.KoiImage;
import java.util.List;
import org.springframework.data.jpa.repository.JpaRepository;

public interface KoiImageRepository extends JpaRepository<KoiImage, Long> {
    List<KoiImage> findByKoiId(Long koiId);
}

File: src/main/java/com/swp391/koibe/repositories/KoiImageRepository.kt
package com.swp391.koibe.repositories

import com.swp391.koibe.models.KoiImage
import org.springframework.data.jpa.repository.JpaRepository

interface KoiImageRepository : JpaRepository<KoiImage?, Long?> {
    fun findByKoiId(koiId: Long?): List<KoiImage?>?
}
